tests/sentry/api/endpoints/test_organization_dashboards.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import absolute_import
+
+from django.core.urlresolvers import reverse
+
+from sentry.models import Dashboard
+from sentry.testutils import APITestCase
+
+
+class OrganizationDashboardsTest(APITestCase):
+    def setUp(self):
+        super(OrganizationDashboardsTest, self).setUp()
+        self.login_as(self.user)
+        self.url = reverse(
+            'sentry-api-0-organization-dashboard',
+            kwargs={'organization_slug': self.organization.slug}
+        )
+
+    def test_get(self):
+        Dashboard.objects.create(
+            title='Dashboard 1',
+            owner=self.user,
+            organization=self.organization,
+        )
+        Dashboard.objects.create(
+            title='Dashboard 2',
+            owner=self.user,
+            organization=self.organization,
+            data={'stuff': 'stuff'}
+        )
+
+        response = self.client.get(self.url)
+        assert response.status_code == 200, response.content
+        assert len(response.data) == 2
+
+        assert response.data[1]['id'] == '1'
```

Comment:
This is just my own perspective on testing that no one else here does, but I think that serializers should be tested separately to endpoints.

Usually what I do is write a test for the serializer and confirm that it produces the expected results. Then on the endpoints, I call the endpoint, then serialize the expected objects and confirm that they match up with what's returned from the endpoint.

No need to change it if you prefer this way, just another perspective on how we could handle these tests.